packages/components/src/components/hds/dropdown/index.hbs

```diff
@@ -13,6 +13,7 @@
       )
     }}
     <div
+      tabindex="-1"
```

Comment:
That's what I noticed the other day when doing the code review, that this `tabindex` was missing: https://github.com/hashicorp/design-system/blob/main/packages/components/src/components/hds/menu-primitive/index.hbs#L16 

(but then I thought we would have noticed if it was important, so I didn't say anything; well, we noticed 🤦‍♂️)